<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentHomeWindow</class>
 <widget class="QDialog" name="StudentHomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student Home</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>631</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>691</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="joinedProjectsList">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>190</y>
     <width>271</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>621</width>
     <height>20</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>631</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="editProfileBtn">
      <property name="text">
       <string>Edit Profile</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="signOutBtn">
      <property name="text">
       <string>Sign Out</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>161</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>        Joined projects</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>381</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Unjoined projects</string>
   </property>
  </widget>
  <widget class="QLabel" name="studentNameLbl">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>619</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Student Name Goes here</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>283</x>
     <y>150</y>
     <width>20</width>
     <height>251</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="statusLbl">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>390</y>
     <width>401</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>[+]Status:</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>190</y>
     <width>391</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QComboBox" name="unjoinedProjectsInput">
      <item>
       <property name="text">
        <string>Select a project to join</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="openProjectBtn">
      <property name="text">
       <string>Open Project</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>joinedProjectsList</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>studentNameLbl</zorder>
  <zorder>line_3</zorder>
  <zorder>statusLbl</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
